packages/bot/src/service/sticker-log.ts

```diff
@@ -0,0 +1,58 @@
+import type { Snowflake } from '../model/id.js';
+import type { StickerEvent, StickerEventResponder } from '../runner/sticker.js';
+import type { StandardOutput } from './output.js';
+
+export interface StickerData {
+  /**
+   * 初期名が自動で割り当てられる絵文字とは異なり,
+   * スタンプは名前を決めてから登録するので名前がuniqueとして有効
+   */
+  stickerName: string; // 絵文字名
+  stickerImageUrl: string; // 絵文字の画像URL
+  stickerId: Snowflake; // 絵文字ID
+  stickerAuthorId: Snowflake; // 絵文字を作成したユーザーID
+  stickerDescription: string; // 絵文字の説明
+  stickerTags: string; // 絵文字のタグ (Autocomplete用)
```

Comment:
```suggestion
  name: string; // 絵文字名
  imageUrl: string; // 絵文字の画像URL
  id: Snowflake; // 絵文字ID
  authorId: Snowflake; // 絵文字を作成したユーザーID
  description: string; // 絵文字の説明
  tags: string; // 絵文字のタグ (Autocomplete用)
```
共通の prefix は冗長なので, 取り除いておきたいです